docs/pages/run/getting-started/installation.md

```diff
@@ -2,7 +2,13 @@
 
 ## Binaries
 
-Binaries can be downloaded from [the release page](https://github.com/ChainSafe/lodestar/releases/latest) under the `Assets` section.
+Binaries can be downloaded from [the release page](https://github.com/ChainSafe/lodestar/releases/latest) under the `Assets` section. You can also run a script that downloads and installs it for you
+
+```bash
```

Comment:
while this does work if a user clones the git repository it is not really convenient and shoudn't be required to install the binaries, it would be best if we host the script on the docs page as noted in my [previous comment](https://github.com/ChainSafe/lodestar/pull/7254/files/6599346376f7acb2af9e443301347353f423474d..5d32426c0836d7af195a78cb24184ba825917736#r1864375801)